intera_examples/scripts/constrained_zeroG.py

```diff
@@ -135,134 +138,112 @@ def main():
     parser.add_argument(
         "-r",  "--rate", type=int, default=10,
         help="A desired publish rate for updating interaction control commands (10Hz by default) -- 0 if we want to publish it only once")
```

Comment:
This is the help that is echoed to the user, so we should add the clarification from above in some capacity:
```
A command of rate 0 means the interaction command is published once and exits. This will cause the arm to remain in interaction control after this script exits.
```